control/src/main/java/org/openbase/bco/dal/control/layer/unit/AbstractUnitController.java

```diff
@@ -1596,4 +1624,192 @@ protected void removeOperationService(final ServiceType serviceType) {
             }
         }
     }
+
```

Comment:
Please declare the following influxdb and query related methods as static and move them to an dedicated util class e.g. `InfluxDbProcessor`